pkg/controllers/provisioning/scheduling/queue.go

```diff
@@ -83,6 +83,19 @@ func byCPUAndMemoryDescending(pods []*v1.Pod) func(i int, j int) bool {
 		} else if memCmp > 0 {
 			return true
 		}
-		return false
+		// If all else is equal, give a consistent ordering. This reduces the number of NominatePod events as we
+		// de-duplicate those based on identical content.
+		lhsPod := pods[i]
```

Comment:
consider setting this at the top of the function, since these helper vars could be used by lines 69/70